core/src/main/java/org/apache/calcite/sql/validate/SqlConformance.java

```diff
@@ -303,6 +303,17 @@ public interface SqlConformance {
    */
   boolean isInsertSubsetColumnsAllowed();
 
+  /**
+   * Whether the dialect is able to ORDER BY literal values.
+   *
+   * This flag is used to filter out literal values in the
+   * order by clause of dialects that don't support them.
+   *
+   * Returns false if the dialect errors when a literal
+   * value is present in ORDER BY, true otherwise.
+   */
```

Comment:
Add a 'Default is false; true in BigQuery' paragraph similar to the other methods in this class.